### DISCUSSION TITLE
Is that possible to compare incoming data with cached data ?

### DISCUSSION THREAD

**Discussion by JohannDesobry** [flutterdata/flutter_data#248] [General]
I have a ranking system and would love to display the number of ranks earned or lost following the updated data.

If it's not possible with flutter_data, I'll handle it with my backend. Thanks for your help 🙏

**Comment by frank06** [comment]
Not sure I understand the problem. You can override `deserialize` on any adapter, and before calling `super.deserialize` can compare what's coming in vs what's in local storage
